classes/search/ArticleSearchIndex.php

```diff
@@ -320,7 +320,7 @@ public function rebuildIndex($log = false, $journal = null, $switches = [])
                     ->getMany();
 
                 foreach ($submissions as $submission) {
-                    if ($submission->getSubmissionProgress() == 0) { // Not incomplete
+                    if ($submission->getSubmissionProgress()) { // Submission not yet submitted
```

Comment:
Did this accidentally invert the test? I think this is expecting to only index completed submissions.

Reply:
Oh good catch! Had to update a bunch of these when going from `int` to `string`.

Although now that I'm looking at it, is this code correct? Should the article search index be checking if the submission is published? Or does this work in both ways (ie - it removes indexing for articles after they have been unpublished)?

Reply:
This is for the command-line tool, which clears the index entirely and then re-generates it from scratch. The check here is to avoid indexing content that is not completely submitted yet.